frontend/src/app/store/migrations/migration-15.ts

```diff
@@ -0,0 +1,20 @@
+export const Decision = {
+  13: (state: any) => {
+    return {
+      ...state,
+      app: {
```

Comment:
could you remove this? as we didn’t change the shape of `state.app` I believe

Reply:
also you're missing to change the version number to 15 in this file `frontend/src/app/store/store.ts`